CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ set_target_properties(stdexec PROPERTIES
                       SOVERSION "${STDEXEC_VERSION_MAJOR}")
 
 # Declare the public include directories
+include(GNUInstallDirs)
 target_include_directories(stdexec INTERFACE
                            $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/>
                            $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

Comment:
`CMAKE_INSTALL_INCLUDEDIR` is used here and defined by `include(GNUInstallDirs)`